sketch_main.cpp

```diff
@@ -16,6 +18,8 @@
 #include <sstream>
 #include <utility>
 
+DEFINE_string(action, "", "Which action to do. One of: triangle");
```

Comment:
Maybe say "One of: triangle, none", and default to none, otherwise this message is quite confusing. 
Also "action" is so generic that it doesn't convey any meaning. Maybe add a quick comment before explaining what action triangle does.